examples/trials/ir_enas/trial_code/src/cifar10/general_child.py

```diff
@@ -0,0 +1,758 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import sys
+
+import numpy as np
+import tensorflow as tf
+
+from src.cifar10.models import Model
+from src.cifar10.image_ops import conv
+from src.cifar10.image_ops import fully_connected
+from src.cifar10.image_ops import batch_norm
+from src.cifar10.image_ops import batch_norm_with_mask
+from src.cifar10.image_ops import relu
+from src.cifar10.image_ops import max_pool
+from src.cifar10.image_ops import global_avg_pool
+
+from src.utils import count_model_params
+from src.utils import get_train_ops
+from src.common_ops import create_weight
+
+
+class GeneralChild(Model):
```

Comment:
This file contains too many lines, I suggest you should split it into several files.